pr_watch/models.py

```diff
@@ -61,7 +61,7 @@ class WatchedFork(models.Model):
         on_delete=models.CASCADE,
         db_index=True,
         help_text=(
-            "Watched GitHub organization. E.g.: open-craft. PRs against the watched fork "
+            "Organization to watch. PRs against against the watched fork."
```

Comment:
@rocioar Ack... You'll need to remove the full stop at the end of this line. Sorry if my [earlier comment](https://github.com/open-craft/opencraft/pull/346#discussion_r212939789) was confusing in that regard?

Also, changing the help text should generate a new migration?

Reply:
@itsjeyd my mistake here, I did it in a rush. Fixed it now, and updated the migrations too (squashed with the other migration)